pkg/distributors/contracts/MultiDistributor.sol

```diff
@@ -235,9 +235,9 @@ contract MultiDistributor is IMultiDistributor, ReentrancyGuard, MultiDistributo
         require(distribution.duration > 0, "DISTRIBUTION_DOES_NOT_EXIST");
         require(distribution.owner == msg.sender, "SENDER_NOT_OWNER");
 
-        // Before receiving the tokens, we must update the distribution's rate
-        // as we are about to change its payment rate, which affects all other rates.
-        _updateDistributionRate(distribution);
+        // Before receiving the tokens, we must sync the distribution up to the present as we are about to change
+        // its payment rate, which would otherwise affect the distribution of tokens distributed since the last update
```

Comment:
> which would otherwise affect the distribution of tokens distributed since the last update

Can this be simplified? It would affect the accounting of tokens distributed since the last update,
or just: "which would otherwise affect the token distribution"

Reply:
I've changed to the first option as I think it's important to describe that we're talking about tokens which have already been distributed (and so should be fixed.)

Reply:
Yes - that's what I was confused about. It can't affect the distribution if they've already been distributed, so I wasn't sure if you mean the accounting of the last, or the future distribution. That clears it up.